src/ncp/posix/infra_if.cpp

```diff
@@ -60,6 +60,19 @@ otbrError InfraIf::Dependencies::SetInfraIf(unsigned int                   aInfr
     return OTBR_ERROR_NONE;
 }
 
+otbrError InfraIf::Dependencies::HandleIcmp6Nd(uint32_t          aInfraIfIndex,
+                                               const Ip6Address &aSrcAddress,
+                                               const uint8_t    *aData,
+                                               uint16_t          aDataLen)
+{
+    OTBR_UNUSED_VARIABLE(aInfraIfIndex);
+    OTBR_UNUSED_VARIABLE(aSrcAddress);
+    OTBR_UNUSED_VARIABLE(aData);
+    OTBR_UNUSED_VARIABLE(aDataLen);
+
+    return OTBR_ERROR_NONE;
+}
```

Comment:
Perhaps we can omit variable names for dummy methods.

@abtink thoughts?

```suggestion
otbrError InfraIf::Dependencies::HandleIcmp6Nd(uint32_t          ,
                                               const Ip6Address &,
                                               const uint8_t    *,
                                               uint16_t          )
{
    return OTBR_ERROR_NONE;
}
```